core/domain/skill_domain.py

```diff
@@ -1439,6 +1439,39 @@ def to_dict(self):
         }
 
 
+class SkillAssignments(python_utils.OBJECT):
+    """Domain object for Skill Assignments, which indicates the topic details to
```

Comment:
Is this for a single topic? If so shouldn't it be SkillAssignment -- or, perhaps better, TopicAssignment (since nothing in this class is about the skill)?

(Remember the principle that the name of the class should describe what's in it and not what it's used for.)

Reply:
Changed to TopicAssignment. 
Done.